OutOfSchool/OutOfSchool.WebApi/Controllers/V1/ChildController.cs

```diff
@@ -142,7 +142,7 @@ public async Task<IActionResult> GetUsersChildById(Guid id)
     [ProducesResponseType(StatusCodes.Status401Unauthorized)]
     [ProducesResponseType(StatusCodes.Status403Forbidden)]
     [ProducesResponseType(StatusCodes.Status500InternalServerError)]
-    [HttpGet("{workshopId}")]
+    [HttpGet("/api/v{version:apiVersion}/workshop/{id}/children/approved/")]
```

Comment:
should be `workshops`.

Also last `/` is not necessary.

Reply:
fixed